packages/mdc-theme/mdc-theme.scss

```diff
@@ -24,7 +24,7 @@
 
 // Special case, so that .mdc-theme--background changes background color, not text color.
 .mdc-theme--background {
```

Comment:
This doesn't look like it's being used or documented anywhere so we can probably get rid of it?

Reply:
We're keeping background. I reverted changes that I made to the components other than chips/card.